com.blizzard.Hearthstone/com.blizzard.Hearthstone.appdata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.blizzard.Hearthstone.desktop</id>
+    <metadata_license>CC0-1.0x</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
+    <name>Hearthstone</name>
+    <!-- <description></description> -->
+    <!-- <screenshots> -->
+    <!-- </screenshots> -->
+    <project_group>winepak</project_group>
+    <developer_name>Blizzard Entertainment</developer_name>
+
+    <url type="homepage">https://playhearthstone.com</url>
+    <url type="help">https://us.battle.net/support/en/games/hearthstone</url>
+    <url type="bugtracker">https://us.battle.net/forums/en/hearthstone/10591462/</url>
+
+    <categories>
+        <category>Game</category>
+        <category>Strategy</category>
+    </categories>
+
+    <content_rating type="oars-1.1">
+        <content_attribute id="violence-fantasy">moderate</content_attribute>
+        <content_attribute id="violence-bloodshed">mild</content_attribute>
+        <content_attribute id="drugs-alcohol">mild</content_attribute>
+        <content_attribute id="sex-themes">mild</content_attribute>
+        <content_attribute id="sex-appearance">moderate</content_attribute>
+        <content_attribute id="social-chat">intense</content_attribute>
+        <content_attribute id="money-purchasing">intense</content_attribute>
+        <content_attribute id="money-gambling">mild</content_attribute>
+    </content_rating>
+
+    <releases>
+        <release date="2018-07-14" version="0.1.0">
```

Comment:
About an hour ago I switched to Calendar Versioning, if you can change it to `2018.07.14` that would be good.